### PR TITLE
Fix some small UI bugs

### PR DESCRIPTION
**Pull request by ustcweizhou**
[UI] fix typo for user data field
[UI] Private Cloud VPC always shows 0 in Site-to-Site VPN, even when one is configured
[UI] change the Icon for changing ACL's for a tier in a VPC
[UI] Network names are not being displayed in Infra » VRs » router » NICS

### CODE REVIEW

**Comment by runseb** [comment]
+1 LGTM, just looked at the code, did not check the actual UI rendering.

File: ui/scripts/vpc.js [hunk 1]
```diff
@@ -4184,7 +4184,7 @@
 
                         // Get VPN gateways
                         $.ajax({
-                            url: createURL('listVpnGateways'),
+                            url: createURL('listVpnGateways&listAll=true'),
                             async: false,
                             data: {
                                 'vpcid': args.context.vpc[0].id
```
Comment (bhaisaab) [inline ui/scripts/vpc.js]: Can you put all the args defined with the url itself in the data object (like vpcid is defined for example, you can add 'listAll': true)

**Comment by wido** [comment]
Based on  the code a LGTM

File: ui/scripts/vpc.js [hunk 2]
```diff
@@ -1507,7 +1507,9 @@
                             },
                             complete: function(args) {
                                 if (args.password != null) {
-                                    alert('Password of the VM is ' + args.password);
+                                    cloudStack.dialog.notice({
+                                        message: 'Password of the VM is ' + args.password
```
Comment (borisroman) [inline ui/scripts/vpc.js]: This I like very much! :)

**Comment by wilderrodrigues** [comment]
Ping @ustcweizhou @remibergsma 

Deploying DC with this PR now. Will submit tests in a moment.

Cheers,
Wilder

**Comment by wilderrodrigues** [comment]
Ping @remibergsma @ustcweizhou 

1. Deployed a DC with a KVM host
2. Created 1 VM under an Isolated Network
3. Create a single VPC with 1 tier, 1 Vm and 1 ACL

Screenshots:

image


But I cannot get the new replace ACL icon. Tried on Chroem, Safari, cleaning the cache and also installed FireFox and tried there. Got the same image:

image

@ustcweizhou: could you please give some hints on how to get it working and just put some screenshots as well? It would also be fine to give the breadcrumbs about how to have the changes you made appearing on our UI.

Cheers,
Wilder

**Comment by borisroman** [comment]
@wilderrodrigues did you remove the .gz files?

File: server/src/com/cloud/api/query/dao/DomainRouterJoinDaoImpl.java [hunk 3]
```diff
@@ -219,6 +219,7 @@ public DomainRouterResponse setDomainRouterResponse(DomainRouterResponse vrData,
             nicResponse.setGateway(vr.getGateway());
             nicResponse.setNetmask(vr.getNetmask());
             nicResponse.setNetworkid(vr.getNetworkUuid());
+            nicResponse.setNetworkName(vr.getNetworkName());
```
Comment (remibergsma) [inline server/src/com/cloud/api/query/dao/DomainRouterJoinDaoImpl.java]: @ustcweizhou Just for my understanding: why do we need this line?

**Comment by karuturi** [comment]
[UI] fix typo for user data field - verfied
verified password dialog change
[UI] Network names are not being displayed in Infra » VRs » router » NICS - Can you share more details about this? probably a screenshot? even without this change, I see network name in nics list

**Comment by ustcweizhou** [comment]
@remibergsma @karuturi 
For the isolated network and shared network, the guest NIC of VR is listed at the top of NIC tab.
However, for VPC VR, the guest NICs are listed at the bottom of NIC tab, the network name is missing for VPC tiers.
Sorry the description of the commit was not clear.

**Comment by wilderrodrigues** [comment]
@ustcweizhou and what about the ACL button?

**Comment by ustcweizhou** [comment]
@wilderrodrigues Please see attachment
image

**Comment by wilderrodrigues** [comment]
Thanks, @ustcweizhou ... this PR LGTM :+1: 

@remibergsma, I believe it's ready to be merged.

Cheers,
Wilder

**Comment by karuturi** [comment]
Thanks @ustcweizhou LGTM :+1:

**Pull request merged**